Modules/_dbmmodule.c

```diff
@@ -24,6 +21,9 @@
     #error "No gdbm/ndbm.h or gdbm-ndbm.h available"
   #endif
   static const char which_dbm[] = "GNU gdbm";
+#elif defined(USE_NDBM)
+  #include <ndbm.h>
+  static const char which_dbm[] = "GNU gdbm";
```

Comment:
I removed the now stale comment about the OS/2 "[EMX](https://en.wikipedia.org/wiki/EMX_(programming_environment)) port" that was added [over 20 years ago](https://github.com/python/cpython/commit/7bf6833e178696b9815fd4e4c6d1cecc8b46bdd2).